src/components/Home/home.js

```diff
@@ -9,12 +9,36 @@ import Box from '@mui/material/Box';
 import AddIcon from '@mui/icons-material/Add';
 import DeleteForeverIcon from '@mui/icons-material/DeleteForever';
 import AccountForm from '../AccountForm/accountForm';
+import { useAuth } from '../../utils/Auth/use-auth';
+import { getAccountsQuery } from '../../services/Account/account';
+import { onSnapshot } from 'firebase/firestore';
+import { financial } from '../../utils/utils/format';
 
 const Home = () => {
-  const [accounts] = useState([]);
+  const auth = useAuth();
+  const user = auth?.data;
+  const [accountsState, setAccountsState] = useState([]);
+  const [selectedAccount, setSelectedAccount] = useState(undefined);
   const [open, setOpen] = useState(false);
-  const handleOpen = () => setOpen(true);
-  const handleClose = () => setOpen(false);
+  const handleOpen = () => setOpen(true)
+  const handleClose = () => { setOpen(false); setSelectedAccount(undefined); }
+
+  useEffect(() => {
+    const accountsQuery = getAccountsQuery(user.uid);
```

Comment:
Minor suggestion: It's a good idead to check if the user.uid has a value to handle when the auth data is still loading so this could be a way to do that
```
  useEffect(() => {
    if(auth.data && !auth.isLoading){
        ...
    }
  }, [auth]);
```